docs/api/requirements.txt

```diff
@@ -1,5 +1,5 @@
 # pip install -r requirements.txt
 
-pydata-sphinx-theme==0.4.0
+pydata-sphinx-theme
```

Comment:
To prevent this from breaking unexpectedly again, I'd prefer to keep this pinned and let Dependabot handle updates (which we can inspect to see if things break). I'll merge the changes from this PR into #497.

Reply:
Ah - thank you.